ci/Dockerfile

```diff
@@ -53,34 +55,15 @@ RUN apt update -yq && apt install -yq software-properties-common \
     libxext6 libxrender1 libxkbcommon-dev libxkbcommon-x11-dev libxcomposite1 libxtst6 \
     libxrandr2 libxcursor1 libxi6 libxcb-randr0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 \
     libxcb-render-util0 libxcb-shape0 libxcb-render0 libxcb-xinerama0 \
-    autoconf automake libtool gcc \
- &&  update-alternatives \
-    --install /usr/bin/gcc gcc /usr/bin/gcc-9 100 \
-    --slave /usr/bin/g++ g++ /usr/bin/g++-9 \
+    autoconf automake libtool gcc g++ \
```

Comment:
Don't we want to lock GCC explicitly? Like `gcc-11` or something?

Reply:
yes 11 is default in this image, I'll pin.
```
jenkins@03b26dd2c7dd:/$ gcc --version
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0

jenkins@03b26dd2c7dd:/$ g++ --version
g++ (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
```